pkg/api/graph/interfaces.go

```diff
@@ -51,6 +53,57 @@ func (m Markers) BySeverity(severity Severity) []Marker {
 	return ret
 }
 
+// FilterCurrentNamespaceMarkers returns all the markers that are not associated with missing nodes
+// from other namespaces (other than the currentNamespace).
+func (m Markers) FilterCurrentNamespaceMarkers(currentNamespace string) Markers {
```

Comment:
`FilterByNamespace(namespace string)`?  It's obviously operating by `Markers` and there's no reason you have to pass in the "current" namespace.